### DISCUSSION TITLE
Разница между general службой и general-батником

### DISCUSSION THREAD

**Discussion by ct2plus** [Flowseal/zapret-discord-youtube#348] [Q&A] [answered]
Большое спасибо за труды.
У меня вопрос: есть ли разумное объяснение тому, что general.bat работает прекрасно, но служба №9 (general), которая, вроде как, "то же самое", невероятно долго загружает видео с ютуба? А также вызывает проблемы со стримингом на ютубе.
иными словами, в качестве службы сервис как будто бы всегда менее стабилен, чем прямой запуск.
P.S. просмотр видео - Firefox, стриминг - OBS. вшитая в ОС админ-учетка, пустой hosts, дом.ру.

**Comment by Flowseal** [comment]
Разницы логически нет. Фактически - возможно косяк сервис инсталлера. Надо смотреть логи, которые пишет service_installer (или смотреть аргументы запуска в самом сервисе).

**Comment by Flowseal** [comment, marked as answer]
Скачайте новую версию (1.5.1). Ещё раз извиняюсь!
